source/incident-collaboration/launching-playbooks.rst

```diff
@@ -1,45 +1,43 @@
 Running Playbooks
 =================
 
-Use slash commands to initiate playbook runs, make announcements, and get information about an incident.
+Use slash commands to initiate playbook runs, make announcements, and get information about a playbook run.
 
 Slash commands
 ~~~~~~~~~~~~~~
 
-The ``/incident`` slash command allows interaction with incidents via the post textbox on desktop, browser, and mobile for:
+The ``/playbook`` slash command allows interaction with incidents via the post textbox on desktop, browser, and mobile for:
 
-- ``/incident start`` - Start a new incident.
-- ``/incident end`` - End an ongoing incident.
-- ``/incident update`` - Update the current incident's status.
-- ``/incident restart`` - Restart an ended incident.
-- ``/incident check [checklist item]`` - Check/uncheck the specified task.
-- ``/incident announce ~[channels]`` - Announce the current incident in other channels.
-- ``/incident list`` - List all your incidents.
-- ``/incident commander [@username]`` - Show or change the current commander.
-- ``/incident info`` - Show a summary of the current incident.
+- ``/playbook start`` - Start a playbook run.
+- ``/playbook end`` - End a playbook run.
+- ``/playbook update`` - Update the current playbook run's status.
+- ``/playbook restart`` - Restart an ended playbook run.
+- ``/playbook check [checklist item]`` - Check/uncheck the specified task.
+- ``/playbook announce ~[channels]`` - Announce the current playbook run in other channels.
+- ``/playbook list`` - List all your running playbooks.
```

Comment:
Is this 100% correct? Because one can have multiple instances of the same playbook running. What about using `ongoing` or something similar?

```suggestion
- ``/playbook list`` - List all your ongoing playbook runs.
```